boot/setup.sh

```diff
@@ -204,6 +210,6 @@ manta_common_setup_end
 #
 # 8881 is the metrics port for manta-garbage-deleter
 #
-mdata-put metricPorts "8881"
+mdata-put metricPorts "8881, ${REBALANCER_AGENT_METRICS_PORT}"
```

Comment:
I don't know if the space would break cmon-agent but in all other places, we don't have any leading space in the csv.